### PR TITLE
Sentinel-3 yaml

### PR DESCRIPTION
**Pull request by Sentinel-5P**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 328d6f84d67d7f7d8855474f3aed9e238d22ff9e
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e52772eb-260d-403b-b83f-33d979808f2f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/sentinel-3.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    Type: S3 Bucket
+  - Description: Sentinel-3 Not Time Critical (NTC) format
+    ARN: arn:aws:s3:::meeo-s3/NTC/
+    Region: eu-central-1
+    Type: S3 Bucket
+  - Description: Sentinel-3 Short Time Critical (STC) format
+    ARN: arn:aws:s3:::meeo-s3/STC/
+    Region: eu-central-1
+    Type: S3 Bucket
+  - Description: Sentinel-3 Cloud Opimised GeoTIFF (COG) format
```
Comment (kylebarron) [inline datasets/sentinel-3.yaml:32]: `Opimised` typo

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 753a7c9b17d5901c0633f126fcb3e940e092e4cd
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=8ae60919-7e85-4f09-ae3c-3bcc4db428e6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: a12c016099649a2371b1e1b7bd7b066d230a1fd5
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=97b68eea-6a2f-441a-a103-6e834efacde3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**